src/test/java/api/loans/scenarios/RefundLostItemFeesTestBase.java

```diff
@@ -293,30 +296,46 @@ public void shouldRefundPartiallyPaidAndTransferredFeeAndCancelRemainingAmount()
 
   @Test
   public void shouldChargeOverdueFineWhenStatedByPolicyAndLostFeesCanceled() {
+    final int overdueMonths = 3;
+    final double overdueFinePerMonth = 10.0;
+    final double expectedOverdueFine = overdueMonths * overdueFinePerMonth;
```

Comment:
Personally, I would prefer this value to be hardcoded as I imagine it is used as an expectation in the test